diff --git a/node_modules/react-native-swipe-list-view/components/SwipeListView.js b/node_modules/react-native-swipe-list-view/components/SwipeListView.js
index 3542801..41a9135 100644
--- a/node_modules/react-native-swipe-list-view/components/SwipeListView.js
+++ b/node_modules/react-native-swipe-list-view/components/SwipeListView.js
@@ -175,10 +175,10 @@ class SwipeListView extends PureComponent {
                     onSwipeValueChange={
                         this.props.onSwipeValueChange
                             ? data =>
-                                  this.props.onSwipeValueChange({
-                                      ...data,
-                                      key,
-                                  })
+                                this.props.onSwipeValueChange({
+                                    ...data,
+                                    key,
+                                })
                             : null
                     }
                     ref={row => (this._rows[key] = row)}
@@ -201,10 +201,10 @@ class SwipeListView extends PureComponent {
                     shouldItemUpdate={
                         this.props.shouldItemUpdate
                             ? (currentItem, newItem) =>
-                                  this.props.shouldItemUpdate(
-                                      currentItem,
-                                      newItem
-                                  )
+                                this.props.shouldItemUpdate(
+                                    currentItem,
+                                    newItem
+                                )
                             : null
                     }
                     setScrollEnabled={enable => this.setScrollEnabled(enable)}
@@ -470,7 +470,11 @@ SwipeListView.propTypes = {
     /**
      * Styles for the parent wrapper View of the SwipeRow
      */
-    swipeRowStyle: ViewPropTypes.style,
+    swipeRowStyle: PropTypes.oneOfType([
+        PropTypes.object,
+        PropTypes.array,
+        PropTypes.number
+    ]),
     /**
      * Called when the FlatList ref is set and passes a ref to the FlatList
      * e.g. listViewRef={ ref => this._swipeListViewRef = ref }
diff --git a/node_modules/react-native-swipe-list-view/components/SwipeRow.js b/node_modules/react-native-swipe-list-view/components/SwipeRow.js
index 05b6fb3..d1019d7 100644
--- a/node_modules/react-native-swipe-list-view/components/SwipeRow.js
+++ b/node_modules/react-native-swipe-list-view/components/SwipeRow.js
@@ -92,7 +92,7 @@ class SwipeRow extends Component {
                 if (
                     !this.isForceClosing &&
                     Dimensions.get('window').width + value <
-                        this.props.forceCloseToRightThreshold
+                    this.props.forceCloseToRightThreshold
                 ) {
                     this.isForceClosing = true;
                     this.forceCloseRow('right');
@@ -111,7 +111,7 @@ class SwipeRow extends Component {
                 if (
                     !this.isForceClosing &&
                     Dimensions.get('window').width - value <
-                        this.props.forceCloseToLeftThreshold
+                    this.props.forceCloseToLeftThreshold
                 ) {
                     this.isForceClosing = true;
                     this.forceCloseRow('left');
@@ -169,9 +169,9 @@ class SwipeRow extends Component {
             dimensionsSet: !this.props.recalculateHiddenLayout,
             ...(!this.props.disableHiddenLayoutCalculation
                 ? {
-                      hiddenHeight: e.nativeEvent.layout.height,
-                      hiddenWidth: e.nativeEvent.layout.width,
-                  }
+                    hiddenHeight: e.nativeEvent.layout.height,
+                    hiddenWidth: e.nativeEvent.layout.width,
+                }
                 : {}),
         });
 
@@ -325,7 +325,7 @@ class SwipeRow extends Component {
                 if (
                     this.currentTranslateX - projectedExtraPixels >
                     this.props.leftOpenValue *
-                        (this.props.swipeToOpenPercent / 100)
+                    (this.props.swipeToOpenPercent / 100)
                 ) {
                     // we're more than halfway
                     toValue = this.isForceClosing
@@ -336,7 +336,7 @@ class SwipeRow extends Component {
                 if (
                     this.currentTranslateX - projectedExtraPixels >
                     this.props.leftOpenValue *
-                        (1 - this.props.swipeToClosePercent / 100)
+                    (1 - this.props.swipeToClosePercent / 100)
                 ) {
                     toValue = this.isForceClosing
                         ? 0
@@ -353,7 +353,7 @@ class SwipeRow extends Component {
                 if (
                     this.currentTranslateX - projectedExtraPixels <
                     this.props.rightOpenValue *
-                        (this.props.swipeToOpenPercent / 100)
+                    (this.props.swipeToOpenPercent / 100)
                 ) {
                     // we're more than halfway
                     toValue = this.isForceClosing
@@ -364,7 +364,7 @@ class SwipeRow extends Component {
                 if (
                     this.currentTranslateX - projectedExtraPixels <
                     this.props.rightOpenValue *
-                        (1 - this.props.swipeToClosePercent / 100)
+                    (1 - this.props.swipeToClosePercent / 100)
                 ) {
                     toValue = this.isForceClosing
                         ? 0
@@ -635,7 +635,11 @@ SwipeRow.propTypes = {
     /**
      * Styles for the parent wrapper View of the SwipeRow
      */
-    style: ViewPropTypes.style,
+    style: PropTypes.oneOfType([
+        PropTypes.object,
+        PropTypes.array,
+        PropTypes.number
+    ]),
     /**
      * Should the row do a slide out preview to show that it is swipeable
      */
